svelte/rollup.config.ts

```diff
@@ -56,7 +56,14 @@ const toRollupConfig = function (component: string): RollupOptions {
     ],
     plugins: [
       svelte({
-        preprocess: [sveltePreprocess({})],
+        preprocess: [
+          sveltePreprocess({
+            sourceMap: !production,
+            postcss: {
+              plugins: [require("autoprefixer")()]
```

Comment:
why use autoprefixer ?

Reply:
It's to permit to use SCSS styles in component, very use full